config/locales/cz.yml

```diff
@@ -225,184 +226,180 @@ cz:
+          _Použij tenhle slider pro úpravu konečné částky. Každá částka nám pomůže! Jestli chceš můžeš se stát pravidelným dárcem, takže se můžeme spolehnout že budeme mít finance i do budoucna._
 
-          CrimethInc. is an all-volunteer project with no outside sources of funding. We offer everything we do for free or at break-even prices while maintaining some of the highest quality standards in our field. We publish constantly, circulating material in [over 30 languages](/tce) on [all seven continents](/2008/01/06/now-serving-seven-continents). We’ve kept this up for a quarter of a century.
+          CrimethInc. je čistě dobrovolnický projekt bez vnějšího financování. Vše co nabízíme, nabízíme zadarmo nebo za ceny na pokrytí nákladů. Stále vydáváme něco nového, a náš materiál je dostupný [ve více než 30 jazycích](/tce) na [všech sedmi kontinentech](/2008/01/06/now-serving-seven-continents). A takhle to děláme už čtvt století.
 
-          We do all this on a shoestring budget—but despite our ingenuity, some expenses are unavoidable. We depend on your assistance to cover those. With more resources, we could expand to utilize more formats and free up time to publish even more frequently.
+          Všechno tohle děláme s minimálním rozpočtem, ale navzdory naší důmyslnosti, se nevyhneme výdajům. A na jejich pokrytí jsme závislí na vaši pomoci. S více zdroji, se můžeme rozšířit na víc formatů a uvolnit čas tak abychom mohli vydávat ještě častěji. 
 
-          As always, of course, the very best way to support what we are doing is to _do it yourself._
+          A jako vždy, nejlepší způsob jak podpořit to co děláme je _D.I.Y._
 
-        radio_label_monthly: Monthly
-        radio_label_one_time: One-time
-        amount_label: Support Amount
-        description_monthly: Monthly Subscription
-        description_one_time: One-Time Support
-        button_text: Next →
+        radio_label_monthly: Měšíční.
+        radio_label_one_time: Jednorázové.
+        amount_label: Částka
+        description_monthly: Měsíční podpora
+        description_one_time: Jednorázová podpora
+        button_text: Další →
         support_request:
-          header: Existing Monthly Subscribers
-          description: Need to update or cancel your subscription? Enter the email address you used to sign up, and we will send you an email with a super secret link where you can make changes.
-          email_label: Email Address
-          email_placeholder: you@example.com
-          button_text: Send Super Secret Link
+          header: Existující měsíční předplatné.
+          description: Potřebuješ upravit nebo zrušit předplatné? Vlož email kterým se přihlašuješ, A my ti na něj pošleme supertajný odkaz kde ho můžeš upravit .
+          email_label: Emailová adresa
+          email_placeholder: ty@příklad.com
```

Comment:
oh this is interesting, `example.com` is actually a [set-aside domain for documentation][0], not sure if we should translate it. @veganstraightedge, thoughts?

[0]:https://en.wikipedia.org/wiki/Example.com